openy.profile

```diff
@@ -799,3 +799,16 @@ function openy_enable_search_api_solr_legacy() {
   return FALSE;
 }
 
+/**
+ * Implements hook_library_info_alter().
+ */
+function openy_library_info_alter(&$libraries, $extension) {
+  // Add jQuery Migrate 4.x to core/jquery library. For more info see https://github.com/jquery/jquery-migrate
+  // This fix is needed for scripts that are not compatible with jQuery 4.x yet.
+  if ($extension == 'core' && isset($libraries['jquery'])) {
```

Comment:
Could you check if version if carnation proper here? https://git.drupalcode.org/project/openy_carnation/-/merge_requests/114

If not correct version - avoid attaching @svicervlad

Reply:
expected version of carnation with fix https://git.drupalcode.org/project/openy_carnation/-/merge_requests/114.

<img width="1224" height="716" alt="Image" src="https://github.com/user-attachments/assets/03e5119b-9f31-4843-a66a-b46b623325c7" />